diff -ruN monodevelop-0.8./monodevelop.in monodevelop-0.8/monodevelop.in
--- monodevelop-0.8./monodevelop.in	2005-10-18 19:49:06.000000000 +0200
+++ monodevelop-0.8/monodevelop.in	2005-10-18 20:30:38.000314250 +0200
@@ -18,12 +18,14 @@
 
 if test -n "$MOZILLA_FIVE_HOME"; then
 	MOZILLA_HOME=$MOZILLA_FIVE_HOME
-elif [ -f @MOZILLA_HOME@/chrome/comm.jar ]; then
-        MOZILLA_HOME=@MOZILLA_HOME@
+elif [ -f @MOZILLA_HOME@/mozilla/chrome/comm.jar ]; then
+        MOZILLA_HOME=@MOZILLA_HOME@/mozilla
 elif [ $(which mozilla) ] && grep MOZILLA_FIVE_HOME= "$(which mozilla)" > /dev/null ; then
 	MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which mozilla) | cut -d '"' -f 2)
 elif [ $(which firefox) ] && grep MOZILLA_FIVE_HOME= "$(which firefox)" > /dev/null ; then
 	MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which firefox) | cut -d '"' -f 2)
+elif [ -f @libdir@/mozilla/chrome/comm.jar ]; then
+	MOZILLA_HOME=@libdir@/mozilla
 else
 	echo "Cannot find mozilla installation directory. Please set MOZILLA_FIVE_HOME to your mozilla directory"
 	exit 1
