--- monodevelop-0.7/configure.orig	2005-05-16 19:18:33.000000000 +0200
+++ monodevelop-0.7/configure	2005-05-19 17:00:50.000000000 +0200
@@ -4446,7 +4446,7 @@
 SQLITE_XML=
 fi
 
-ALL_LINGUAS="da de es fr ja_JP pt_BR tr pl"
+ALL_LINGUAS="da de es fr ja pt_BR tr pl"
 GETTEXT_PACKAGE=monodevelop
 
 
--- monodevelop-0.7/configure.in.orig	2005-05-15 22:11:04.000000000 +0200
+++ monodevelop-0.7/configure.in	2005-05-19 17:02:38.000000000 +0200
@@ -153,7 +153,7 @@
 fi
 
 dnl Intl
-ALL_LINGUAS="da de es fr ja_JP pt_BR tr pl"
+ALL_LINGUAS="da de es fr ja pt_BR tr pl"
 GETTEXT_PACKAGE=monodevelop
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
