diff -ur monodevelop-0.5.orig/configure monodevelop-0.5/configure
--- monodevelop-0.5.orig/configure	2004-06-29 21:19:39.000000000 +0200
+++ monodevelop-0.5/configure	2004-09-19 23:50:43.000000000 +0200
@@ -20182,7 +20182,7 @@
 fi
 
 
-ALL_LINGUAS="da es fr ja_JP pt_BR"
+ALL_LINGUAS="da es fr ja pt_BR"
 GETTEXT_PACKAGE=monodevelop
 
 
diff -ur monodevelop-0.5.orig/configure.in monodevelop-0.5/configure.in
--- monodevelop-0.5.orig/configure.in	2004-09-19 23:25:42.000000000 +0200
+++ monodevelop-0.5/configure.in	2004-09-19 23:50:48.000000000 +0200
@@ -119,7 +119,7 @@
 AM_CONDITIONAL(ENABLE_XDGMIME, test "x$enable_xdgmime" = "xyes")
 
 dnl Intl
-ALL_LINGUAS="da es fr ja_JP pt_BR"
+ALL_LINGUAS="da es fr ja pt_BR"
 GETTEXT_PACKAGE=monodevelop
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
