--- monodevelop-0.18/monodevelop.in~	2007-12-19 16:47:09.004304700 +0100
+++ monodevelop-0.18/monodevelop.in	2007-12-19 16:48:28.498841222 +0100
@@ -16,42 +16,7 @@
     echo "$ret"
 }
 
-
-#(NOTE: any changes to this block should be kept in sync with the one in configure.in)
-MOZFILE="libgtkembedmoz.so"
-find_libgtkembedmoz ()
-{
-	MOZDIR=$MOZILLA_FIVE_HOME
-	if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-	
-	MOZDIR=$(grep -h GRE_PATH= /etc/gre.d/*.conf 2>/dev/null | cut -d '"' -f 2 -d = | head -n 1) #"
-	if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-	
-	mozilla_script=$(which mozilla 2> /dev/null)
-	firefox_script=$(which firefox 2> /dev/null)
-	
-	if [ -z $mozilla_script ] && [ -z $firefox_script ]; then return; fi
-	for runtime_script in "$firefox_script $mozilla_script"; do
-		MOZDIR=$(grep "MOZ_DIST_LIB=" $runtime_script 2> /dev/null | cut -d '"' -f 2 | cut -d '=' -f 2)
-		if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-		MOZDIR=$(grep "MOZILLA_FIVE_HOME=" $runtime_script 2> /dev/null | cut -d '"' -f 2 | cut -d '=' -f 2)
-		if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-		MOZDIR=$(grep "MOZILLA_LIBDIR=" $runtime_script 2> /dev/null | cut -d '"' -f 2 | cut -d '=' -f 2)
-		if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-		MOZDIR=$(grep "moz_libdir=" $runtime_script 2> /dev/null | cut -d '"' -f 2 | cut -d '=' -f 2)
-		if test -e "$MOZDIR/$MOZFILE"; then echo $MOZDIR; return; fi
-	done
-}
-
-if test -e "@MOZILLA_HOME@/$MOZFILE"; then
-	MOZILLA_HOME="@MOZILLA_HOME@"
-else
-	MOZILLA_HOME=`find_libgtkembedmoz`
-	if test ! -e "$MOZILLA_HOME/$MOZFILE"; then
-		MOZILLA_HOME=
-		echo "WARNING: Cannot find Mozilla directory containing $MOZFILE. Some Addins may not be able to function. Please set MOZILLA_FIVE_HOME to your Mozilla directory."
-	fi
-fi
+MOZILLA_HOME=@libdir@/xulrunner
 
 MD_BIN_PATH=@prefix@/lib/monodevelop/bin
 
