--- monodevelop-0.6/monodevelop.desktop.orig	2005-03-21 20:49:17.000000000 +0100
+++ monodevelop-0.6/monodevelop.desktop	2005-05-13 23:48:17.000000000 +0200
@@ -3,11 +3,11 @@
 Encoding=UTF-8
 Name=MonoDevelop
 GenericName=Integrated Development Environment
+GenericName[pl]=Zintegrowane ¶rodowisko programistyczne
 Comment=Integrated Development Environment
 Exec=monodevelop
 TryExec=monodevelop
 Icon=monodevelop.png
-StartupNotify=true
 Terminal=false
 Type=Application
 MimeType=text/x-csharp;application/x-mds;application/x-mdp;application/x-cmbx;application/x-prjx;
