diff -aurN monodevelop-0.9.orig/monodevelop.desktop monodevelop-0.9/monodevelop.desktop
--- monodevelop-0.9.orig/monodevelop.desktop	2005-09-23 22:11:15.000000000 +0200
+++ monodevelop-0.9/monodevelop.desktop	2005-11-30 20:29:02.000000000 +0100
@@ -3,15 +3,16 @@
 Encoding=UTF-8
 Name=MonoDevelop
 GenericName=Integrated Development Environment
+GenericName[pl]=Zintegrowane Środowisko Programistyczne
 Comment=Integrated Development Environment
 Exec=monodevelop
 TryExec=monodevelop
-Icon=monodevelop.png
+Icon=monodevelop
 StartupNotify=true
 Terminal=false
 Type=Application
 MimeType=text/x-csharp;application/x-mds;application/x-mdp;application/x-cmbx;application/x-prjx;
-Categories=Application;Development;
+Categories=GTK;GNOME;Development;IDE;
 X-GNOME-Bugzilla-Bugzilla=Ximian
 X-GNOME-Bugzilla-Product=MonoDevelop
 X-GNOME-Bugzilla-OtherBinaries=monodevelop
