--- monodevelop/configure.in~	2007-12-19 16:50:59.488464049 +0100
+++ monodevelop/configure.in	2007-12-19 16:52:25.982519510 +0100
@@ -280,7 +280,7 @@
 AC_SUBST(CSC_FLAGS)
 
 
-MD_DIR='$(prefix)/lib/monodevelop'
+MD_DIR='$(libdir)/monodevelop'
 MD_ASSEMBLY_DIR="$MD_DIR/bin"
 MD_ADDIN_DIR="$MD_DIR/AddIns"
 
--- monodevelop/Makefile.am~	2007-12-19 16:50:59.488464049 +0100
+++ monodevelop/Makefile.am	2007-12-19 16:52:55.980457820 +0100
@@ -15,7 +15,7 @@
 
 pkgconfig_in_files = monodevelop.pc.in monodevelop-core-addins.pc.in
 
-pkgconfigdir= $(prefix)/lib/pkgconfig
+pkgconfigdir= $(libdir)/pkgconfig
 pkgconfig_DATA = $(pkgconfig_in_files:.pc.in=.pc)
 
 if ENABLE_UPDATE_MIMEDB
--- monodevelop/mdtool.in~	2007-12-18 20:11:34.000000000 +0100
+++ monodevelop/mdtool.in	2007-12-19 16:53:48.476849862 +0100
@@ -16,11 +16,10 @@
     echo "$ret"
 }
  
-MD_BIN_PATH=@prefix@/lib/monodevelop/bin
+MD_BIN_PATH=@libdir@/monodevelop/bin
 
 ### SYNCH BLOCK: any changes to this block should be kept in sync with the one in Makefile.include and monodevelop.in
-MD_PKG_CONFIG_PATH=@gtksharp_prefix@/lib/pkgconfig/:/usr/lib/pkgconfig/:/usr/local/lib/pkgconfig/:/usr/share/pkgconfig/:/usr/local/share/pkgconfig/
-if test -d /usr/lib64; then MD_PKG_CONFIG_PATH=$MD_PKG_CONFIG_PATH:/usr/lib64/pkgconfig/:/usr/local/lib64/pkgconfig/; fi
+MD_PKG_CONFIG_PATH=@libdir@/pkgconfig/
 ### END BLOCK
 
 if [ -n $PKG_CONFIG_PATH ]; then
--- monodevelop/monodevelop.pc.in~	2007-12-18 20:11:34.000000000 +0100
+++ monodevelop/monodevelop.pc.in	2007-12-19 16:54:14.475063064 +0100
@@ -1,6 +1,6 @@
 prefix=@prefix@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib/monodevelop
+libdir=@libdir@
                                                                                 
 Name: MonoDevelop
 Description: Free .NET Development Environment
