diff -Nuard monodevelop-0.5.orig/monodevelop.in monodevelop-0.5/monodevelop.in
--- monodevelop-0.5.orig/monodevelop.in	2004-06-27 06:03:48.000000000 +0200
+++ monodevelop-0.5/monodevelop.in	2004-07-04 08:45:50.673470032 +0200
@@ -27,7 +27,7 @@
 	exit 1
 fi
  
-MD_BIN_PATH=@prefix@/lib/monodevelop/bin
+MD_BIN_PATH=@libdir@/monodevelop/bin
 
 if [ -n $LD_LIBRARY_PATH ]; then
 	export LD_LIBRARY_PATH=$MOZILLA_HOME:$MD_BIN_PATH:$LD_LIBRARY_PATH
@@ -36,9 +36,9 @@
 fi
 
 if [ -n $PKG_CONFIG_PATH ]; then
-	export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:@gtksharp_prefix@/lib/pkgconfig/
+	export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:@libdir@/pkgconfig/
 else
-	export PKG_CONFIG_PATH=@gtksharp_prefix@/lib/pkgconfig/
+	export PKG_CONFIG_PATH=@libdir@/pkgconfig/
 fi
 
 export MOZILLA_FIVE_HOME=$MOZILLA_HOME
