diff -ur monodevelop-0.10/monodevelop.desktop monodevelop-0.10-desktop_patch/monodevelop.desktop
--- monodevelop-0.10/monodevelop.desktop	2006-04-04 21:29:37.000000000 +0200
+++ monodevelop-0.10-desktop_patch/monodevelop.desktop	2006-04-05 18:13:56.000000000 +0200
@@ -3,7 +3,9 @@
 Encoding=UTF-8
 Name=MonoDevelop
 GenericName=Integrated Development Environment
+GenericName[pl]=Zintegrowane Środowisko Programistyczne
 Comment=Develop .NET applications in an Integrated Development Environment
+Comment[pl]=Rozwijaj aplikacje .NET w Zintegrowanym Środowisku Programistycznym
 Exec=monodevelop
 TryExec=monodevelop
 Icon=monodevelop
