--- monodevelop-0.11/monodevelop.in.orig	2006-05-06 01:26:55.000000000 +0200
+++ monodevelop-0.11/monodevelop.in	2006-06-07 08:28:04.000000000 +0200
@@ -18,12 +18,16 @@
 
 if test -n "$MOZILLA_FIVE_HOME"; then
 	MOZILLA_HOME=$MOZILLA_FIVE_HOME
-elif [ -f @MOZILLA_HOME@/chrome/comm.jar ]; then
-        MOZILLA_HOME=@MOZILLA_HOME@
-elif [ $(which mozilla) ] && grep MOZILLA_FIVE_HOME= "$(which mozilla)" > /dev/null ; then
-	MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which mozilla) | cut -d '"' -f 2)
-elif [ $(which firefox) ] && grep MOZILLA_FIVE_HOME= "$(which firefox)" > /dev/null ; then
-	MOZILLA_HOME=$(grep MOZILLA_FIVE_HOME= $(which firefox) | cut -d '"' -f 2)
+elif [ -f @MOZILLA_HOME@/mozilla/chrome/comm.jar ]; then
+        MOZILLA_HOME=@MOZILLA_HOME@/mozilla
+elif [ -f @libdir@/mozilla/chrome/comm.jar ]; then
+	MOZILLA_HOME=@libdir@/mozilla
+elif [ -f @libdir@/seamonkey/chrome/comm.jar ]; then
+	MOZILLA_HOME=@libdir@/seamonkey
+elif [ -f @libdir@/mozilla-firefox/chrome/comm.jar ]; then
+	MOZILLA_HOME=@libdir@/mozilla-firefox
+elif [ -f @libdir@/mozilla-thunderbird/chrome/comm.jar ]; then
+	MOZILLA_HOME=@libdir@/mozilla-thunderbird
 else
 	echo "Cannot find mozilla installation directory. Please set MOZILLA_FIVE_HOME to your mozilla directory"
 	exit 1
