diff -ur monodevelop-0.10/monodevelop.desktop monodevelop-0.10-desktop_patch/monodevelop.desktop
--- monodevelop-0.10/monodevelop.desktop	2006-04-04 21:29:37.000000000 +0200
+++ monodevelop-0.10-desktop_patch/monodevelop.desktop	2006-04-05 18:13:56.000000000 +0200
@@ -3,15 +3,17 @@
 Encoding=UTF-8
 Name=MonoDevelop
 GenericName=Integrated Development Environment
+GenericName[pl]=Zintegrowane ¦rodowisko Programistyczne
 Comment=Develop .NET applications in an Integrated Development Environment
+Comment[pl]=Rozwijaj aplikacje .NET w Zintegrowanym ¦rodowisku Programistycznym
 Exec=monodevelop
 TryExec=monodevelop
-Icon=monodevelop.png
+Icon=monodevelop
 StartupNotify=true
 Terminal=false
 Type=Application
 MimeType=text/x-csharp;application/x-mds;application/x-mdp;application/x-cmbx;application/x-prjx;
-Categories=Application;Development;
+Categories=GTK;GNOME;Development;IDE;
 X-GNOME-Bugzilla-Bugzilla=Ximian
 X-GNOME-Bugzilla-Product=MonoDevelop
 X-GNOME-Bugzilla-OtherBinaries=monodevelop
Tylko w monodevelop-0.10-desktop_patch: monodevelop.desktop~
