--- monodevelop-0.11/configure~	2006-05-06 11:17:14.686161250 +0200
+++ monodevelop-0.11/configure	2006-05-06 11:20:03.672722250 +0200
@@ -4619,7 +4619,7 @@
 fi
 
 
-ALL_LINGUAS="cs da de es fr ja_JP pt_BR tr pl it zh_TW zh_CN ru"
+ALL_LINGUAS="cs da de es fr ja pt_BR tr pl it zh_TW zh_CN ru"
 GETTEXT_PACKAGE=monodevelop
 
 
--- monodevelop-0.11/configure.in~	2006-05-06 11:17:14.690161500 +0200
+++ monodevelop-0.11/configure.in	2006-05-06 11:19:41.883360500 +0200
@@ -190,7 +190,7 @@
    AM_CONDITIONAL(ENABLE_UPDATE_DESKTOPDB, test x$enable_update_desktopdb = xyes)
 
 dnl Intl
-ALL_LINGUAS="cs da de es fr ja_JP pt_BR tr pl it zh_TW zh_CN ru"
+ALL_LINGUAS="cs da de es fr ja pt_BR tr pl it zh_TW zh_CN ru"
 GETTEXT_PACKAGE=monodevelop
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
