--- monodevelop-0.6/configure.orig	2005-03-21 20:50:28.000000000 +0100
+++ monodevelop-0.6/configure	2005-05-14 00:09:25.000000000 +0200
@@ -22200,7 +22200,7 @@
 SQLITE_XML=
 fi
 
-ALL_LINGUAS="da es fr ja_JP pt_BR tr"
+ALL_LINGUAS="da es fr ja pt_BR tr"
 GETTEXT_PACKAGE=monodevelop
 
 
--- monodevelop-0.6/configure.in.orig	2005-03-21 20:49:17.000000000 +0100
+++ monodevelop-0.6/configure.in	2005-05-14 00:08:31.000000000 +0200
@@ -183,7 +183,7 @@
 fi
 
 dnl Intl
-ALL_LINGUAS="da es fr ja_JP pt_BR tr"
+ALL_LINGUAS="da es fr ja pt_BR tr"
 GETTEXT_PACKAGE=monodevelop
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
